tests/runtime-tests.sh

```diff
@@ -17,4 +17,4 @@ echo "bpftrace --info:"
 echo "===================="
 "${BPFTRACE_RUNTIME_TEST_EXECUTABLE}/bpftrace" --info
 
-python3 runtime/engine/main.py $@
+python3 runtime/engine/main.py "$@"
```

Comment:
I think we actually want the word splitting here. Otherwise if there were multiple args then main would incorrectly only see 1 parameter, right?

Reply:
Without double quotes, cmd like : ./tests/runtime-tests.sh --test-filter "probe.kprobe_wildcard probe builtin" throws error:

main.py: error: unrecognized arguments: probe builtin

Without quotes works fine for test names which doesn't contain "space" or "dash", otherwise fail due to command split at space/dash.

With quotes, multiple args passed as single string to python works fine.

Reply:
Ah, I see. "$@" is special cased: https://stackoverflow.com/questions/3811345/how-to-pass-all-arguments-passed-to-my-bash-script-to-a-function-of-mine

Reply:
Ah, I see. "$@" is special cased: https://stackoverflow.com/questions/3811345/how-to-pass-all-arguments-passed-to-my-bash-script-to-a-function-of-mine